config/routes.rb

```diff
@@ -80,6 +80,7 @@
         get :choose_meters, to: 'energy_tariffs#choose_meters'
         get :default_tariffs, to: 'energy_tariffs#default_tariffs'
         get :smart_meter_tariffs, to: 'energy_tariffs#smart_meter_tariffs'
+        get :group_school_tariffs, to: 'energy_tariffs#group_school_tariffs'
```

Comment:
don't think there was any specific design decision.  perhaps just a carry over from the old tariff editor or for readability?  but i think as it's out of scope of this pr this can go in a separate ticket to clean up the routes and add some additional tests.

Reply:
OK no probs, yes agree out of scope - happy to create a ticket 👍